src/getting-started/assets/index.html

```diff
@@ -4,14 +4,14 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="Content-Security-Policy" content="default-src 'unsafe-inline'">
-  <title>Java Getting Started</title>
+  <title>Coding and Debugging Tips for Students</title>
```

Comment:
Change it to "for Beginners"? Since this is also useful to new VS Code user.

Reply:
good suggestion

Reply:
updated.